src/main/kotlin/uk/gov/gdx/datashare/models/CognitoClientResponse.kt

```diff
@@ -1,12 +1,15 @@
 package uk.gov.gdx.datashare.models
 
+import com.fasterxml.jackson.annotation.JsonInclude
 import io.swagger.v3.oas.annotations.media.Schema
 
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(description = "Cognito Client Response")
 data class CognitoClientResponse(
-  @Schema(description = "Cognito client name", required = true, example = "HMPO")
+  @Schema(description = "Cognito client name", required = true, example = "HMPO", maxLength = 80, pattern = "^[a-zA-Z0-9-_]{1,50}\$")
```

Comment:
Are these max lengths all educated approximations and upper bounds? Or do they come from somewhere?

Reply:
We should check if AWS API has an upper bound length